pkg/apis/portworx/register.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package portworx
+
+const (
+	// GroupName is the group name for the portworx CRD
+	GroupName = "portworx.com"
```

Comment:
Might want to use portworx.org or portworx.io instead

Reply:
I kept portworx.com since it's resolvable. portworx.org or portworx.io will be dead links.

Reply:
That doesn't need to be resolvable, and we should probably get portworx.io anyways

Reply:
https://github.com/portworx/talisman/pull/69